.github/workflows/tests.yml

```diff
@@ -0,0 +1,22 @@
+name: tests
+on:
+  push:
+    branches-ignore:
+      - 'master'
+
+jobs:
+  tests:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 1
+      - uses: actions/setup-node@v1
+        with:
+          node-version: '8.x'
```

Comment:
probably could go higher than 8 -- think 8 was just the highest LTS at the time the wercker file was set up?

Reply:
I'll work on upgrading node. I tried and it seemed like some things broke so I'll put it in another pull request 👍